static/favicons/safari-pinned-tab.svg

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="310.000000pt" height="310.000000pt" viewBox="0 0 310.000000 310.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.14, written by Peter Selinger 2001-2017
```

Comment:
will this cause any legal issues?

Reply:
Hi Arhell, unfortunately we can't use this tool -- there is no OSS license for the source code, and potrace 1.14 uses the GPL license, which we have a policy against using in the project.